chapters/dae.tex

```diff
@@ -9,75 +9,68 @@ \chapter{Modelica DAE Representation}\doublelabel{modelica-dae-representation}
 used components by:
 \begin{itemize}
 \item
-  expanding all class definitions (flattening the inheritance tree) and
+  Expanding all class definitions (flattening the inheritance tree) and
   adding the equations and assignment statements of the expanded classes
   for every instance of the model.
 \item
-  replacing all connect-equations by the corresponding equations of the
+  Replacing all connect-equations by the corresponding equations of the
   connection set (see \autoref{generation-of-connection-equations}).
 \item
-  mapping all algorithm sections to equation sets.
+  Mapping all algorithm sections to equation sets.
 \item
-  mapping all when-clauses to equation sets (see \autoref{when-equations}).
+  Mapping all when-clauses to equation sets (see \autoref{when-equations}).
 \end{itemize}
 
 As a result of this transformation process, a set of equations is
 obtained consisting of differential, algebraic and discrete equations of
-the following form where ($v:=\lbrack \dot{x}; x; y; t; m; \text{pre}(m); p\rbrack$ ):
+the following form where ($v := \lbrack \dot{x}; x; y; t; m; \text{\lstinline!pre!}(m); p\rbrack$):
 \begin{subequations}
 \begin{equation}
-c:=f_c(relation(v))\label{eq:crossing}
+c:=f_{\mathrm{c}}(\mathit{relation}(v))\label{eq:crossing}
 \end{equation}
 \begin{equation}
-\dot{m}:=f_m(v, v)
+\dot{m}:=f_m(v, c)
```

Comment:
Good change - I don't know how that error occurred, but if we are anyway clearing up that equation the `\dot{m}` should be replaced by just `m`

Reply:
Fixed.